### PR TITLE
Improve deploy board to make health check results clear

### PR DESCRIPTION
**Pull request by liyaqin1**
Screenshot 2023-05-12 at 4 05 02 PM

### CODE REVIEW

File: deploy-board/deploy_board/templates/groups/health_check_activities.tmpl [hunk 1]
```diff
@@ -42,6 +42,10 @@
                     <span class="deployToolTip pointer-cursor {{ health_check.status | healthCheckStatusIcon }}"></span>
                     {% if health_check.status == "SUCCEEDED" %}
                         STATE SUCCEEDED
+                    {% elif health_check.status == "QUALIFIED" %}
+                        QUALIFIED (PENDING_TERMINATION)
+                    {% elif health_check.status == "TELETRAAN_STOP_REQUESTED" %}
+                        QUALIFIED (TERMINATED)
```
Comment (tylerwowen) [inline deploy-board/deploy_board/templates/groups/health_check_activities.tmpl:48]: Technically it's not necessary terminated.

Comment (osoriano) [inline deploy-board/deploy_board/templates/groups/health_check_activities.tmpl:48]: :+1:

**Review by osoriano — DISMISSED** [review]

File: deploy-board/deploy_board/templates/groups/health_check_activities.tmpl (hunk 1 above)
Comment (osoriano) [reply deploy-board/deploy_board/templates/groups/health_check_activities.tmpl:48]: ^ @liyaqin1 
Agreed, maybe just `QUALIFIED`?

Comment (liyaqin1) [reply deploy-board/deploy_board/templates/groups/health_check_activities.tmpl:48]: I don't see the difference between terminated and TELETRAAN_STOP_REQUESTED. If we want a strict terminated, we need to add a status in rodimus side. From my point of view, TELETRAAN_STOP_REQUESTED is the last status in the whole process. If you want me to update in rodimus side, feel free to let me know.

**Review by tylerwowen — APPROVED** [review]

**Pull request merged**